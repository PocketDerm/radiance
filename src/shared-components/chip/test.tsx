import React from 'react';
import renderer from 'react-test-renderer';

import { Chip } from './index';

describe('<Chip />', () => {
  describe('UI snapshots', () => {
    it('renders the correct css and text', () => {
      const component = renderer.create(
        <Chip status="success" text="Success" />,
      );

      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
