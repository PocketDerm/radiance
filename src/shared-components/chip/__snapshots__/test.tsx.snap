// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chip /> UI snapshots renders the correct css and text 1`] = `
.emotion-2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 4px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 1.5rem;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 0.5rem;
  background-color: #2B6E33;
}

.emotion-2 .emotion-1 {
  color: #FFFFFF;
}

.emotion-0 {
  color: #524D6E;
  font-size: 0.75rem;
  line-height: 1.67;
  font-weight: bold;
  -webkit-transform: translateY(1px);
  -ms-transform: translateY(1px);
  transform: translateY(1px);
}

<div
  class="emotion-2 emotion-3"
>
  <span
    class="emotion-0 emotion-1"
  >
    Success
  </span>
</div>
`;
